src/Components/Header/Header.css

```diff
@@ -41,14 +41,14 @@ Make it a variable... how?
 
   /* H5/ Roboto Regular 24 */
   float: right;
-  font-family: Tungsten-Semibold;
+  font-family: Roboto;

```

Comment:
The change to Tungsten was from this task: https://www.notion.so/uwblueprintexecs/Update-the-Arbitrium-logo-font-to-Tungsten-34b6b413ff5c4291a92282d8cd8802fb

Was there a reason it was changed back to Roboto?

Reply:
during QA we noticed that the font wasn't displaying properly (I think it was showing as Times New Roman), so we changed it back to Roboto as a quick fix to release the app. let's definitely try to get the Tungsten font back in though!